### DISCUSSION TITLE
LinkedIn error: Cannot display preview. You can post as is, or try another link.

### DISCUSSION THREAD

**Discussion by haytham-mostafa** [forem/forem#20474] [Q&A]
Hi all,

I am facing the mentioned error during posting a new article in LinkedIn : https://dev.to/haythammostafa/aws-lambda-triggers-and-event-sources-2kbl

After investigation, I have found this error message in my article page: 
`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://stats.g.doubleclick.net/j/collect?t=dc&aip=1&_r=3&v=1&_v=j101&tid=UA-71991109-1&cid=481562154.1680756069&jid=783302251&gjid=531443255&_gid=905017462.1702900694&_u=SACAAEAAEAAAACAAI~&z=982288468. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 499`

Please advise.

Thanks

**Comment by Shubham00043** [comment]
This might be happening because LinkedIn’s crawler is unable to access the preview metadata correctly due to a temporary CORS or access restriction issue.

You can try:

* clearing LinkedIn post cache by reposting the link
* testing the article in an incognito window
* checking whether the article is publicly accessible without login
* waiting some time and retrying, since LinkedIn preview generation sometimes fails temporarily

The `Access-Control-Allow-Origin` message usually points to a cross-origin restriction while LinkedIn tries to fetch preview data.
